pkg/pillar/types/zedmanagertypes.go

```diff
@@ -264,13 +264,6 @@ func RoundupToKB(b uint64) uint64 {
 	return (b + 1023) / 1024
 }
 
-// ErrorInfo errorInfo holder structure
-type ErrorInfo struct {
-	Error       string
-	ErrorSource string
-	ErrorTime   time.Time
-}
-
```

Comment:
I fail to understand the value of moving this definition from the types package to the types package.

Reply:
Oh, you shouldn't eve be using it since it leads to confusion about what ErrorSource is about and since you 1) don't need it 2) don't know what it is for in zedmanager and volumemgr, we end up with unneeded complexity.

Reply:
Its a generic error structure that can be used in any place you need to store Error information. You don;t need 2 / 3 fields to be added to every structure - but instead use this..

Since this is generic - I thought a more generic place like types.go is better than retaining in zedrouter or zedmanager types.